src/components/UsergroupList.jsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import style from "../scss/UsergroupList.scss";
+
+export default class UsergroupList extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      usergroups: <i className="ld ld-ring ld-cycle" />
+    };
+    this.getUsergroups();
+  }
+
+  getUsergroups = () => {
+    const siteUrl = this.props.siteUrl;
+    const that = this;
```

Comment:
dlaczego? I tak korzystasz z tego w funkcji strzałkowej która ma "poprawne" this